client/src/components/Home.js

```diff
@@ -5,17 +5,24 @@ import { Button } from "semantic-ui-react";
 class Home extends React.Component {
   constructor(props) {
     super(props);
+    this.joinRoom = this.joinRoom.bind(this);
   }
 
   componentDidMount() {
     console.log(this.props)
   }
 
+  joinRoom = () => {
+    this.props.socket.emit('joinRoom', {
+      roomHash: this.props.roomHashDuo
+    })
+  }
+
   render() {
     return (
       <div>
         <h1>Home Page</h1>
-        <Button>
+        <Button onClick={this.joinRoom}>
```

Comment:
Instead of nesting `<Link>` in `<Button`> component, You could have added `className="ui button"` to `Link` only and `onClick` event seprately.